src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -39,9 +39,13 @@ The .NET Foundation licenses this file to you under the MIT license.
       <CrossCompileArch Condition="$(CrossCompileRid.EndsWith('-x64'))">x86_64</CrossCompileArch>
       <CrossCompileArch Condition="$(CrossCompileRid.EndsWith('-arm64')) and '$(_IsApplePlatform)' != 'true'">aarch64</CrossCompileArch>
       <CrossCompileArch Condition="$(CrossCompileRid.EndsWith('-arm64')) and '$(_IsApplePlatform)' == 'true'">arm64</CrossCompileArch>
+      <CrossCompileArch Condition="$(CrossCompileRid.EndsWith('-arm'))">arm</CrossCompileArch>
+
+      <CrossCompileAbi>gnu</CrossCompileAbi>
+      <CrossCompileAbi Condition="$(CrossCompileRid.EndsWith('-arm'))">gnueabihf</CrossCompileAbi>
 
       <TargetTriple />
-      <TargetTriple Condition="'$(CrossCompileArch)' != ''">$(CrossCompileArch)-linux-gnu</TargetTriple>
+      <TargetTriple Condition="'$(CrossCompileArch)' != ''">$(CrossCompileArch)-linux-$(CrossCompileAbi)</TargetTriple>
```

Comment:
@am11 care to review this part?

Reply:
This looks good and neat. Other related/interesting triplets are:

```
arm-linux-androideabi
arm-unknown-linux-gnueabi
arm-unknown-linux-gnueabihf
arm-unknown-linux-musleabi
arm-unknown-linux-musleabihf
armv7-unknown-linux-gnueabi
armv7-unknown-linux-gnueabihf
armv7-unknown-linux-musleabi
armv7-unknown-linux-musleabihf
```

of which, maybe `<CrossCompileAbi Condition="'$(CrossCompileRid)' == 'linux-musl-arm'">musleabi</CrossCompileAbi>` could be added? We have prereq docker image available for it as well.

---
Side note:
AFAIK, triplet is considered an archaic concept by some toolchain enthusiasts. They recommend using explicit `-march`, `-mtune`, `-mabi`, `-mcpu` and `-mfpu` options. At some point we can delve into that and bring nativeaot and mono targets to new age. :)

Reply:
I guess I'll have some follow up PRs later on, so I can add the "musl" then.

Reply:
No problem, take your time. Basically the next line (49) can be deleted after that, since `alpine` is unnecessary and more restrictive than `unknown` or not setting the vendor at all; which we have for gnu and bionic.